.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,41 @@
+name: Create Release Pull Request
```

Comment:
For monorepos we're using `create-release-branch` nowadays instead of `action-create-release-pr`, as the former allows for independent releases (like releasing one package, but not the other).

You can take a look at the `core` repo for an example of how to set this up. The most important parts are here:

- https://github.com/MetaMask/core/blob/8ad3072b472d8b4a5e36dd31802dcf4f4fd646a3/docs/contributing.md#releasing
- https://github.com/MetaMask/core/blob/8ad3072b472d8b4a5e36dd31802dcf4f4fd646a3/release.config.json
- https://github.com/MetaMask/core/blob/8ad3072b472d8b4a5e36dd31802dcf4f4fd646a3/.github/workflows/main.yml#L27-L49
- https://github.com/MetaMask/core/blob/8ad3072b472d8b4a5e36dd31802dcf4f4fd646a3/.github/workflows/publish-release.yml

I _think_ `create-release-pr` still works with monorepos, but I would recommend switching to `create-release-branch`.

Reply:
Thanks for the hint, I'll take a look. The monorepo is left over from the snap template so I can also simplify the repo layout if that makes more sense.

Reply:
@Mrtenz I've adapted the workflows to consider the monorepo layout.
The only package that matters is the `snap`. The `site` package is just for development.

If you think it's wise, I can refactor the repo to no longer use a monorepo after the 2 open PRs are merged

Reply:
@Mrtenz now the repository is refactored into a regular repo